lib/humboldt/cli.rb

```diff
@@ -60,7 +61,19 @@ def run_local
       end
       input_glob = File.join(options[:data_path], options[:input])
       hadoop_config_path = options[:hadoop_config] || default_hadoop_config_path
-      run_command('hadoop', 'jar', project_jar, '-conf', hadoop_config_path, job_config, input_glob, output_path, *options[:extra_hadoop_args])
+      hadoop_command = ['hadoop', 'jar', project_jar]
+      hadoop_config = ['-conf', hadoop_config_path]
+      job_args = [input_glob, output_path, *options[:extra_hadoop_args]]
+      command = hadoop_command.dup
+      if Rubydoop::VERSION > '2.0.0'
```

Comment:
My line above doesn't handle a version like `2.0.0.pre0`. Perhaps better to stick with string comparison and just change the operator, or compare with something like `2.0.0.a`.